Applications/Browser/BrowserConsoleClient.cpp

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) 2018-2020, Hunter Salyer <thefalsehonesty@gmail.com>
```

Comment:
Just 2020 probably ? (it's the same for all the files)

Reply:
Ah good catch, copy-paste strikes again!